src/Administration/Resources/app/administration/src/app/filter/currency.filter.js

```diff
@@ -1,8 +1,10 @@
+import { types } from 'src/core/service/util.service';
+
 const { Filter } = Shopware;
 const { currency } = Shopware.Utils.format;
```

Comment:
```suggestion
const { Filter } = Shopware;
const { types } = Shopware.Utils;
const { currency } = Shopware.Utils.format;
```

Using imports like that shouldn't be done anymore :)